CGATPipelines/Pipeline/Execution.py

```diff
@@ -368,8 +368,8 @@ def run(**kwargs):
     options.update(kwargs.items())
 
     # insert a few legacy synonyms
-    options['cluster_options'] = options.get('job_options',
-                                             options['cluster_options'])
+    options['cluster_options'] += " " + options.get('job_options',
+                                                    "")
```

Comment:
So, what is the difference between `job_options` and `cluster_options`? Please point me to any docs that I may have missed.

The original statement:
```
options['cluster_options'] = options.get('job_options', options['cluster_options'])
```
gives higher priority to `job_options` over `cluster_options` in case they are both defined. If that is not right, we should just remove it and therefore `cluster_options` will not be overwritten.

I assume we do not want both sets of options to be mixed in a single place, like:
```
options['cluster_options'] += options.get('job_options')
```
which is semantically equivalent to this PR's proposal but cleaner syntax. Then, it would be up to the job scheduler to decide what to do with redundant options and may cause different results on different job schedulers.